docs/src/user-docs/guides-quick-start.md

```diff
@@ -158,7 +173,7 @@ create a database and authorize a user to access it.
 To build the scheduler, run:
 
 ```shell
-cmake --build build --parallel $(nproc) --target spider_scheduler
+cmake --build build/spider --parallel $(nproc) --target spider_scheduler
```

Comment:
There are some instance that were missed. E.g., the one below should be `build/spider/spider`. I'd recommend going through the quick-start guide and making sure every command is correct.